src/core/border_router/routing_manager.cpp

```diff
@@ -3367,7 +3429,7 @@ void RoutingManager::PdPrefixManager::ProcessPlatformGeneratedRa(const uint8_t *
     Error                                     error = kErrorNone;
     Ip6::Nd::RouterAdvertMessage::Icmp6Packet packet;
 
-    VerifyOrExit(mEnabled, LogWarn("Ignore platform generated RA since PD is disabled."));
+    VerifyOrExit(IsRunning(), LogWarn("Ignore platform generated RA since PD is disabled or not running."));
```

Comment:
Should this log be at info level? This seems like it is expected behavior.
```suggestion
    VerifyOrExit(IsRunning(), LogInfo("Ignore platform generated RA since PD is disabled or not running."));
```

Reply:
I think this should be LogWarn, as we are not expecting the platform generating RA messages for us.

If the users disabled routing manager or pd in thread stack, they should also disable the  related feature on the platform.